example/basic/main.go

```diff
@@ -16,25 +16,45 @@ package main
 
 import (
 	"context"
+	"log"
 
 	"go.opentelemetry.io/otel/api/distributedcontext"
 	"go.opentelemetry.io/otel/api/key"
 	"go.opentelemetry.io/otel/api/metric"
 	"go.opentelemetry.io/otel/api/trace"
+	"go.opentelemetry.io/otel/exporter/trace/stdout"
 	"go.opentelemetry.io/otel/global"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
 )
 
 var (
-	tracer = global.TraceProvider().GetTracer("ex.com/basic")
-	meter  = global.MeterProvider().GetMeter("ex.com/basic") // TODO: should share resources ^^^?
-
 	fooKey     = key.New("ex.com/foo")
 	barKey     = key.New("ex.com/bar")
 	lemonsKey  = key.New("ex.com/lemons")
 	anotherKey = key.New("ex.com/another")
 )
 
+// initTracer creates and registers trace provider instance.
+func initTracer() {
+	var err error
+	exp, err := stdout.NewExporter(stdout.Options{PrettyPrint: false})
+	if err != nil {
+		log.Panicf("failed to initialize stdout exporter %v\n", err)
+		return
+	}
+	tp, err := sdktrace.NewProvider(sdktrace.WithSyncer(exp),
+		sdktrace.WithConfig(sdktrace.Config{DefaultSampler: sdktrace.AlwaysSample()}))
+	if err != nil {
+		log.Panicf("failed to initialize trace provider %v\n", err)
+	}
+	global.SetTraceProvider(tp)
+}
+
 func main() {
+	initTracer()
+	tracer := global.TraceProvider().GetTracer("ex.com/basic")
+	meter := global.MeterProvider().GetMeter("ex.com/basic") // TODO: should share resources ^^^?
```

Comment:
please add a note that Meter doesn't work yet.

Reply:
Just rephrase TODO, thanks.

Reply:
FYI there is a Meter stdout exporter in #265, which also adds to this example.
It's fine to remove the note about flush.